### PR TITLE
aps spider: add authorization to requests

### PR DESCRIPTION
**Pull request by MJedr**
Ref: inspirehep/inspirehep#1669

### CODE REVIEW

File: hepcrawl/spiders/aps_spider.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ def __init__(self, from_date=None, until_date=None,
         self.until_date = until_date
         self.per_page = per_page
         self.date = date
+        self.aps_token = aps_token
```
Comment (michamos) [inline hepcrawl/spiders/aps_spider.py:66]: maybe store the authorization header instead, as that's what you're using everywhere?
```suggestion
        self.auth_header = "Bearer {}".format(aps_token)
```

**Review by michamos — APPROVED** [review]
small suggestion, but looks ok

File: hepcrawl/spiders/aps_spider.py [hunk 2]
```diff
@@ -85,7 +86,7 @@ def url(self):
     def start_requests(self):
         """Just yield the url."""
         started_at = datetime.utcnow()
-        yield Request(self.url)
+        yield Request(self.url, headers={self.auth_header})
```
Comment (michamos) [inline hepcrawl/spiders/aps_spider.py:89]: typo? this should be a `dict`

Comment (MJedr) [reply hepcrawl/spiders/aps_spider.py:89]: Thank you!

**Pull request merged**